cucoslib/workers/gh_metadata_result_collector.py

```diff
@@ -0,0 +1,32 @@
+from selinon import StoragePool
+from cucoslib.base import BaseTask
+from cucoslib.utils import get_analysis_by_id
+
+
+class GitHubManifestMetadataResultCollector(BaseTask):
+    """
+    Collect all results that were computed, upload them to S3 and store version reference to results in WorkerResult
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def run(self, arguments):
+        self._strict_assert(arguments.get('ecosystem'))
+        self._strict_assert(arguments.get('repo_name'))
```

Comment:
Yeah, I saw it. Maybe we would like to unify storing by parsing url as with the current approach there could be created for example `www.github.com` and `github.com` S3 "folder" - not sure about the data from jobs though (note that the flow can be scheduled even manually).